content/news/vcmi-1.1.0-release.md

```diff
@@ -0,0 +1,38 @@
++++
+categories = ["Releases", "Stable"]
+date = "2022-12-23"
+description = ""
+featured = "vcmirelease1_1.jpg"
+featuredalt = ""
+featuredpath = "date"
+linktitle = ""
+title = "VCMI 1.1.0 released"
+type = "post"
+forumurl = "https://forum.vcmi.eu/"
+
++++
+
+## Big update before 2023!
+ 
+Three months passed since our previous release and we observed many downloads and quite big interest to the project from the community.
+We want to say thank you for that and willing for more! Such support realy motivates us so we were able to increase our efforts towards new features and would like to share with you one more big update this year.
+
+## iOS support
+VCMI supports one more platfform! Now you can play heroes on your iPad or iPhone and it'll have all game mechanics, including mods. Installation is not so simple as it could be using embedded app store but it definetely worth to make several simple steps from this [instruction](https://wiki.vcmi.eu/Installation_on_iOS) and it will be easier than expected!
```

Comment:
```suggestion
VCMI supports one more platform! Now you can play heroes on your iPad or iPhone and it'll have all game mechanics, including mods. Installation is not so simple as it could be using the official AppStore but it's definitely worth to make several simple steps from this [instruction](https://wiki.vcmi.eu/Installation_on_iOS) and it will be easier than expected!
```